src/decorators/json/index.js

```diff
@@ -39,10 +39,10 @@ export const jsonDecorator = makeDecorator(
         const rawValue = this.asModel._getRaw(rawFieldName)
         const parsedValue = parseJSON(rawValue)
 
-        return sanitizer(parsedValue)
+        return sanitizer.call(this, parsedValue)
```

Comment:
I'm not comfortable with passing the Modal as `this`... Feels too magical. Why not just add second argument?

Reply:
Yes, agreed! Just pushed the changes.